### ISSUE TITLE
Unusable on latest CFW/OFW

### ISSUE THREAD

**Issue by PKPenguin321** [J-D-K/Avatool#6]
Edit: Caused by the 9.0 update as far as I can tell.

On latest Atmosphere/OFW, the app does not respond to button inputs and cannot be used.

**Comment by PKPenguin321** [comment]
A usable compile by the repo owner was found here: https://gbatemp.net/threads/avatool-homebrew-tool-to-change-avatars.525654/page-3#post-8906548

**Comment by Suffice** [comment]
I tried the one in the link, and it seemed to work as intended (It'd crash in about 20 seconds if it succeeded). However now I can't get it to work on my GF's switch at all anymore. I keep getting this error: attempting to mount 0x800...010... Failed. The only thing I did to change things was add another account. Deleting the account doesn't appear to solve the problem. Version 11.0.1.

Update: I was able to get it to work with Goldleaf (had to launch out of applet mode and select the right account, the desired image in 256x256 jpg, and then restart the system for it to take effect).
